mirdata/datasets/compmusic_otmm_makam.py

```diff
@@ -0,0 +1,235 @@
+    MusicBrainz. The makam and the tonic of each recording is annotated in the file annotations.json.
+
+    The audio related data in the test dataset is organized by each makam in the folder data. Due to copyright reasons,
+    we are unable to distribute the audio. Instead we provide the predominant melody of each recording, computed by a
+    state-of-the-art predominant melody extraction algorithm optimized for OTMM culture. These features are saved as
+    text files (with the paths data/[makam]/[mbid].pitch) of single column that contains the frequency values. The
+    timestamps are removed to reduce the filesizes. The step size of the pitch track is 0.0029 seconds (an analysis
+    window of 128 sample hop size of an mp3 with 44100 Hz sample rate), with which one can recompute the timestamps of
+    samples.
+
+    Moreover the metadata of each recording is available in the repository, crawled from MusicBrainz using an open
+    source tool developed by us. The metadata files are saved as data/[makam]/[mbid].json.
+
+    For reproducability purposes we note the version of all tools we have used to generate this dataset in the
+    file algorithms.json (not integrated in the loader but present in the donwloaded dataset).
+
+    A complementary toolbox for this dataset is MORTY, which is a mode recogition and tonic identification toolbox.
+    It can be used and optimized for any modal music culture. Further details are explained in the publication above.
+"""
+
+import csv
+import json
+import os
+from typing import TextIO
+
+import numpy as np
+from mirdata import annotations, core, download_utils, io, jams_utils
+
+BIBTEX = """
+@software{sertan_senturk_2016_58413,
+  author       = {Sertan Şentürk and
+                  Altuğ Karakurt},
+  title        = {{otmm_makam_recognition_dataset: Ottoman-Turkish
+                   Makam Music Makam Recognition Dataset}},
+  month        = jul,
+  year         = 2016,
+  publisher    = {Zenodo},
+  version      = {dlfm2016},
+  doi          = {10.5281/zenodo.58413},
+  url          = {https://doi.org/10.5281/zenodo.58413}
+}
+"""
+
+REMOTES = {
+    "all": download_utils.RemoteFileMetadata(
+        filename="otmm_makam_recognition_dataset-dlfm2016.zip",
+        url="https://zenodo.org/record/58413/files/otmm_makam_recognition_dataset-dlfm2016.zip?download=1",
+        checksum="c2b9c8bdcbdcf15745b245adfc793145",
+    )
+}
+
+LICENSE_INFO = (
+    "Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License"
+)
+
+
+class Track(core.Track):
+    """OTMM Makam Track class
+
+    Args:
+        track_id (str): track id of the track
+        data_home (str): Local path where the dataset is stored. default=None
+            If `None`, looks for the data in the default directory, `~/mir_datasets`
+
+    Attributes:
+        makam (str): string referring to the makam represented in the track
```

Comment:
tiny nit - pitch_paths and mb_tags_paths are missing from this docstring

Reply:
True!!!! Sorry, haven't noticed. I'll fix it now.
By the way, `tonic`, `makam` and `mbid` are properties now. Should I add a `Properties` section in the docstring and put them there?
```
Properties:
    tonic (float): bla bla bla
    makam (str): bla bla bla
    ...
Catched properties:
    pitch (F0Data): bla bla bla
```